static/js/components/UserProfileInfo.jsx

```diff
@@ -5,9 +5,43 @@ import Card from "@material-ui/core/Card";
 import CardContent from "@material-ui/core/CardContent";
 import Typography from "@material-ui/core/Typography";
 import Box from "@material-ui/core/Box";
+import PropTypes from "prop-types";
 
 import UserAvatar from "./UserAvatar";
 
+export const UserContactInfo = ({ user }) => {
+  let contact_string = "";
+  if (user.first_name || user.last_name) {
+    contact_string += `${user.first_name} ${user.last_name}`;
```

Comment:
```suggestion
    contact_string += `${user.first_name} ${user.last_name} `;
```
Space to put space between name/username and contact info